tests/csapi/apidoc_room_create_test.go

```diff
@@ -61,6 +61,72 @@ func TestRoomCreate(t *testing.T) {
 			})
 			content := alice.MustGetStateEventContent(t, roomID, "m.room.topic", "")
 			must.MatchGJSON(t, content, match.JSONKeyEqual("topic", "Test Room"))
+
```

Comment:
It looks like we have to gate the tests that expect `m.topic` so that Dendrite doesn't run them https://github.com/matrix-org/complement/actions/runs/16165352807/job/45668238300?pr=788#step:8:1971

For this one, I guess we just have to split it out separately.